lib/main.coffee

```diff
@@ -31,9 +26,19 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-phpmd.rulesets',
       (rulesets) =>
         @rulesets = rulesets
-    @subscriptions.add atom.config.observe 'linter-phpmd.projectRules',
-      (projectRules) =>
-        @projectRules = projectRules
+        if atom.config.get('linter-phpmd.projectRules') is true
```

Comment:
No worries. I lose notifications all the time, especially when I'm overwhelmed with greenkeeper whenever a bunch of modules gets updated… 👋